__tests__/operation.test.ts

```diff
@@ -30,6 +31,10 @@ describe('#getSummary()', () => {
   it('should return nothing if not present', () => {
     expect(Oas.init(referenceSpec).operation('/2.0/users/{username}', 'get').getSummary()).toBeUndefined();
   });
+
+  it('should allow a common summary to override the operation-level summary', () => {
+    expect(Oas.init(parametersCommon).operation('/anything/{id}', 'get').getSummary()).toBe('[common] Summary');
```

Comment:
This honestly feels a lil counterintuitive to me... I would think that since the operation-level summary is more specific, it should override the common summary 🤔 but if that's the expected behavior within the OAS, fine by me 🤷🏽

Reply:
Agreed. It makes sense for `parameters` since if you have a path param you don't really want to dupe that on every path item, but for summary or description? ehhh